src/python/pybind11/src/pyxrt.cpp

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright (C) 2016-2020 Xilinx, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+#include <iostream>
+#include <stdexcept>
+#include <string>
+#include <cstring>
+#include <tuple>
+#include <typeinfo>
+
+// XRT includes
+#include "experimental/xrt_device.h"
+#include "experimental/xrt_kernel.h"
+#include "experimental/xrt_bo.h"
+#include "xrt.h"
```

Comment:
Is this include needed here? Using symbols out of xrt.h may imply the need to link with xrt_core.  This in turn complicates using python code with emulation.   If we rely on coreutil.so to load the shim (core) library then emulation automatically works.

Reply:
removed xrt.h in 2ccb6f8